stat/distuv/triangle.go

```diff
@@ -194,6 +194,22 @@ func (t Triangle) Score(deriv []float64, x float64) []float64 {
 	return deriv
 }
 
+// ScoreInput returns the score function with respect to the input of the
+// distribution at the input location specified by x. The score function is the
+// derivative of the log-likelihood
+//  (d/dx) log(p(x)) .
+// Special cases:
+//  ScoreInput(t.c) = NaN
```

Comment:
`.c` never shows up in the godoc, so this should probably just be `c`, possibly with a reminder that c is the mode.

Is it worth noting it is not defined outside of (a, b)?

Reply:
It is. Done. PTAL.